examples/policygen/config.yaml

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+template_dir: ../../templates/policygen
```

Comment:
This and org_id are required, could you leave a comment marking them as such?

Reply:
Done